homework-1/css/style.css

```diff
@@ -0,0 +1,1220 @@
+}
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  margin: 0;
+  padding: 0;
+}
+ul,
+li {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+button {
+  padding: 0;
+  cursor: pointer;
+  border: none;
+  outline: none;
+  background: transparent;
+}
+body {
+  min-width: 375px;
+  overflow-x: hidden;
+  font-family: "Aclonica-Regular", sans-serif;
+  font-weight: 400;
+  margin: 0;
+  padding: 0;
+  position: relative;
+}
+body.lock {
+  overflow: hidden;
+}
+.container {
+  padding-left: 20px;
+  padding-right: 20px;
+  margin: 0 auto;
+  min-width: 375px;
+}
+.hidden {
+  display: none !important;
+}
+.logo__link {
+  font-family: "Actor-Regular";
+  color: #fff;
+  font-size: 26px;
+  line-height: 26px;
+  text-transform: uppercase;
+  font-weight: 400;
+  font-weight: bold;
+  letter-spacing: 1.2px;
+}
+.header {
+  z-index: 1;
+  padding: 18px 20px;
+}
+.header .menu__list {
+  display: flex;
+  flex-direction: column;
+}
+.header .menu__list > li {
+  position: relative;
+  margin: 0 auto;
+}
+.header .menu__link {
+  font-size: 12px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  color: #22BF95;
+}
+.header .header__button-link {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+  top: 80px;
+}
+.header .header__button-link a {
+  font-size: 12px;
+  border-radius: 20px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+  color: #22BF95;
+  padding: 8px 0;
+  border: 1.5px solid #22BF95;
+  border-radius: 20px;
+}
+.header .header__banner {
+  height: 211px;
+}
+.header .menu__body {
+  position: absolute;
+  top: 64px;
+  left: 0;
+  width: 100%;
+  background-color: #fff;
+  transition: transform 0.5s ease-out;
+  z-index: 1;
+  transform: translateY(-120vh);
+}
+.header .menu__body.active {
+  transform: translateY(0px);
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+.header .menu__icon {
+  display: block;
+  width: 24px;
+  height: 18px;
+  position: relative;
+  cursor: pointer;
+}
+.header .menu__icon::before, .header .menu__icon::after, .header .menu__icon span {
+  position: absolute;
+  left: 0;
+  height: 10%;
+  width: 100%;
+  background-color: #fff;
+  transition: all 0.3s ease 0s;
+}
+.header .menu__icon::before, .header .menu__icon::after {
+  content: "";
+}
+.header .menu__icon::before {
+  top: 0;
+}
+.header .menu__icon::after {
+  bottom: 0;
+}
+.header .menu__icon span {
+  top: 50%;
+  transform: scale(1) translate(0px, -50%);
+}
+.header .menu__icon.active span {
+  transform: scale(0) translate(0px, -50%);
+}
+.header .menu__icon.active::before {
+  top: 50%;
+  transform: rotate(-45deg) translate(0px, -50%);
+}
+.header .menu__icon.active::after {
+  bottom: 50%;
+  transform: rotate(45deg) translate(0px, 50%);
+}
+.header .get__link {
+  width: 180px;
+}
+.header .log__link {
+  width: 90px;
```

Comment:
Please, avoid ardcoded width. Use max-width instead and width: 100%;

Reply:
done